### PR TITLE
Bug Fix for -BypassListForUsers & Doc accuracy

### PR DESCRIPTION
**Pull request by Romanitho**
# Proposed Changes

> Doc precision on features

### CODE REVIEW

**Review by KnifMelti — COMMENTED** [review]
I don't follow you, what am I supposed to review?

File: README.md [hunk 1]
```diff
@@ -60,55 +60,52 @@ A new Auto-Update process has been released from version 1.5.0. By default, WAU
...
 Use White List instead of Black List. This setting will not create the "excluded_apps.txt" but "included_apps.txt".
 
-**-ListPath**  
-Get Black/White List from external Path (**URL/UNC/Local/GPO**) - download/copy to Winget-AutoUpdate installation location if external list is newer.  
-**PATH** must end with a Directory, not a File...  
+**-ListPath**<br>
+Get Black/White List from external Path (**URL/UNC/Local/GPO**) - download/copy to Winget-AutoUpdate installation location if external list is newer.<br>
+**PATH** must end with a Directory, not a File...<br>
 ...if the external Path is an **URL** and the web host doesn't respond with a date/time header for the file (i.e **GitHub**) then the file is always downloaded!
+If `-ListPath` is set to **GPO** the Black/White List can be managed from within the GPO itself under **Application GPO Blacklist**/**Application GPO Whitelist**. Thanks to [Weatherlights](https://github.com/Weatherlights) in [#256 (reply in thread)](https://github.com/Romanitho/Winget-AutoUpdate/discussions/256#discussioncomment-4710599)!
```
Comment (KnifMelti) [inline README.md:104]: <br> before

File: README.md [hunk 2]
```diff
@@ -117,87 +114,86 @@ For **URL**: This requires a site directory with **Directory Listing Enabled** a
...
 For **AzureBlob**: This requires the parameter **-AzureBlobURL** to be set with an appropriate Azure Blob Storage URL including the SAS token. See **-AzureBlobURL** for more information.
 
-**-AzureBlobURL**
+**-AzureBlobURL**<br>
 Used in conjunction with the **-ModsPath** parameter to provide the Azure Storage Blob URL with SAS token. The SAS token must, at a minimum, have 'Read' and 'List' permissions. It is recommended to set the permisions at the container level and rotate the SAS token on a regular basis. Ensure the container reflects the same structure as found under the initial `mods` folder. (From version 1.16.4).
 
-**-InstallUserContext**  
-Install WAU with system and **user** context executions (From version 1.15.3).
+**-InstallUserContext**<br>
+Install WAU with system and **user** context executions (From version 1.15.3). Applications installed in system context will be ignored under user context.
```
Comment (KnifMelti) [inline README.md:129]: `<br>` before:

>  Applications installed in...

File: README.md [hunk 3]
```diff
@@ -117,87 +114,86 @@ For **URL**: This requires a site directory with **Directory Listing Enabled** a
...
 ## Custom scripts (Mods feature)
 From version 1.8.0, the Mods feature allows you to run additional scripts when upgrading or installing an app.
-Just put the scripts in question with the **AppID** followed by the `-preinstall`, `-upgrade`, `-install` or `-installed` suffix in the **mods** folder.  
-> Runs before upgrade/install: `AppID-preinstall.ps1`  
-> Runs during upgrade/install (before install check): `AppID-upgrade.ps1`/`AppID-install.ps1`  
-> Runs after upgrade/install has been confirmed: `AppID-installed.ps1`  
+Just put the scripts in question with the **AppID** followed by the `-preinstall`, `-upgrade`, `-install` or `-installed` suffix in the **mods** folder.
+
+> Runs before upgrade/install: `AppID-preinstall.ps1`<br>
+Runs during upgrade/install (before install check): `AppID-upgrade.ps1`/`AppID-install.ps1`<br>
```
Comment (KnifMelti) [inline README.md:169]: Maybe `>-`

File: README.md [hunk 4]
```diff
@@ -117,87 +114,86 @@ For **URL**: This requires a site directory with **Directory Listing Enabled** a
...
+Runs after upgrade/install has been confirmed: `AppID-installed.ps1`
```
Comment (KnifMelti) [inline README.md:170]: Maybe `>-`

File: README.md [hunk 5]
```diff
@@ -117,87 +114,86 @@ For **URL**: This requires a site directory with **Directory Listing Enabled** a
...
-The **GPO ADMX/ADML** validated with:  
-[Windows 10 - Validate ADMX for Ingestion](https://developer.vmware.com/samples/7115/windows-10---validate-admx-for-ingestion)  
+In an enterprise environment it's crucial that different groups can have different settings in applications etc. or to implement other mandatory settings, i.e for security/management reasons.<br>
+**WAU** doesn't have any setting that can be changed except for when installing (or editing the registry/the task `Winget-AutoUpdate` as **Admin**).<br>
+With the use of **ADML/ADMX** files you can manage every **WAU** setting from within **GPO**.<br>
+They will be detected/evaluated during the next run of **WAU** (taking effect before any actions).<br>
+The **GPO ADMX/ADML** validated with: [Windows 10 - Validate ADMX for Ingestion](https://developer.vmware.com/samples/7115/windows-10---validate-admx-for-ingestion)<br>
 Read more in the `README.md` under the directory **Policies**.
 
 ![image](https://user-images.githubusercontent.com/102996177/213920242-7ff8e2b4-d926-4407-b860-1e5922e29c3e.png)
```
Comment (KnifMelti) [inline README.md:199]: New image when you're done with your GPO setting

**Comment by Romanitho** [comment]
That's exactly why I need you 👌🏻😊

File: README.md (hunk 1 above)
Comment (Romanitho) [reply README.md:104]: Maybe we could also remove 3 dots :/

**Pull request merged**